spec/order_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'order'
+
+describe Order do
+  let(:description) {"Cheeseburger"}
+  let(:price) {4.99}
+  let(:burger) { double :dish, :description => description,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.